app/SomeClass.php

```diff
@@ -0,0 +1,15 @@
+<?php
+final class SomeClass
+{
+    private int $count;
+
+    public function __construct()
+    {
+        $this->count = 123;
+    }
+
+    public function getCount():int
+    {
+        return $this->count;
+    }
+}
```

Comment:
This file is probably left-over from local testing, should be removed

Reply:
Didn't realize, however, that I should remove the app testing code and will do so going forward.